docs/content/testing/running-tests.md

```diff
@@ -0,0 +1,62 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../../README.md#recommended-use).
+
+-----
+
+# Running tests
+
+Run the following commands in your terminal from the root of the repository.
+
+For the first run, you need these commands:
+
+```bash
+./prepare-workspace.sh
+./ci/build-test.sh
+```
+
+If you have modified anything in the code generator or docs linter tool;
+ to re-test these you need to re-run this command:
+
+```bash
+./ci/build-test.sh
+```
+
+If you have modified anything in the Unity Project; to re-test this you can
+ re-run this command:
+
+```bash
+./ci/test.sh
+```
+
+## Unity tests
+
+For Unity tests, you can also run tests from the Unity Test Runner Window.
+
+### 1. Open the Unity project in Unity 
```

Comment:
Is this the GDK Unity project, rather than a user's Unity project? It isn't clear.

Reply:
"1. Open the Unity GDK project in the Unity Editor"